docs/security/trust/trust_key_mng.md

```diff
@@ -11,19 +11,34 @@ parent= "smn_content_trust"
 # Manage keys for content trust
 
 Trust for an image tag is managed through the use of keys. Docker's content
-trust makes use four different keys:
+trust makes use of five different types of keys:
```

Comment:
I see this was already there, but is there a reason to have a description of the five keys in two places? (Here, and in content_trust.md)